packages/owl-bot/cloud-build/update-pr.yaml

```diff
@@ -12,6 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This cloud build gets invoked every time a new pull request is created or
+# a new commit is added to a pull request that's being watched by Owl Bot.
+# It's triggered programmatically by event handlers in Owl Bot.
+# It runs the post processor specified in the environment variable 
+
+##############################################################################
```

Comment:
I wonder, could we have a step run conditionally ([there seem to be some hacky ways to do so](https://stackoverflow.com/questions/58235945/google-cloud-build-conditional-step)) and that way eliminate the need for a second cloud build file?

Reply:
I don't think the hacky solutions can be applied to our situation.